docs/install/2-install-server-with-docker.rst

```diff
@@ -305,42 +257,40 @@ There are two ways you can achieve this:
             image: containers.inmanta.com/containers/service-orchestrator:|version_major|
             ports:
                 - 8888:8888
+            env_file: ./resources/my-env-file
+            environment:
+                INMANTA_DATABASE_HOST: inmanta_db
+                INMANTA_DATABASE_USERNAME: inmanta
+                INMANTA_DATABASE_PASSWORD: inmanta
             volumes:
                 - ./resources/com.inmanta.license:/etc/inmanta/license/com.inmanta.license
                 - ./resources/com.inmanta.jwe:/etc/inmanta/license/com.inmanta.jwe
                 - ./resources/my-server-conf.cfg:/etc/inmanta/inmanta.cfg
-                - ./resources/my-env-file:/etc/inmanta/env
 
+Mounting files/directories
+########################
 
-Changing inmanta user/group id
-##############################
+The recommended way to mount files and directories is to use docker volumes:
 
-If you mount a folder of your host in the container, and expect the inmanta user to interact with it,
-you might face the issue that the inmanta user inside the container doesn't have ownership of the files.
-You could fix this by changing the ownership in the container, but this change would also be reflected
-on the host, meaning that you would lose the ownership of you files.  This is a very uncomfortable
-situation.
-While ``Podman`` has been offering the possibility to do a mapping of a user id in the container to a
-user id on the host at runtime, which would solve our problem here, ``Docker`` still doesn't offer this
-functionality.
-The inmanta container allows you to change the user and group id of the inmanta user inside the
-container when starting the container to match the user on the host, getting rid that way of any
-conflict in the files ownership.
+.. code-block:: sh
 
-This can be done easily by simply setting those environment variables:
- - ``INMANTA_UID``: Will change, when starting the container, the id of the inmanta user.
- - ``INMANTA_GID``: Will change, when starting the container, the id of the inmanta group.
+    docker volume create mydockervolume
```

Comment:
I would assume that docker-compose has support to create volumes itself, so that you don't have to execute this command. I know it has support to delete volumes, so then it would surprise me if it wouldn't be able to create them.

Reply:
Indeed I forgot about it. I will modify this